python-lib/gluonts_forecasts/custom_models/utils.py

```diff
@@ -1,14 +1,20 @@
 def cast_kwargs(kwargs):
```

Comment:
Nitpick, but these functions would be better located in a top-level utils.py, close to the function to parse lists and stuff.

Reply:
these functions are only used by the custom models and specific to them.

Reply:
oki